cluster/gce/gci/configure-helper.sh

```diff
@@ -1010,6 +1010,21 @@ function start-kube-apiserver {
 
 
   local authorization_mode="RBAC"
+
+  # Create the ABAC file only if it's explicitly requested.
+  if [[ -n "${ENABLE_LEGACY_ABAC}" ]]; then
```

Comment:
I'd like a rename to `ENABLE_LEGACY_ABAC_16_ONLY` and and an issue to delete this in 1.7.